tests/lib/tools/lxd-state

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh -e
 case "${1:-}" in
+    -h|--help|'')
+        echo "usage: lxd-state [-h] {undo-lxd-mount-changes} ..."
+        ;;
     undo-lxd-mount-changes)
```

Comment:
unrelated but maybe the command can skip having lxd in it and be just "undo-mount-changes"

Reply:
Yeah, good point.

Reply:
Since this is +3, I will do it in a follow-up.